frontend/src/httpClient.js

```diff
@@ -1,30 +1,27 @@
 import axios from 'axios';
 
-export default class HttpClient {
+const backendBaseUrl = process.env.REACT_APP_BACKEND_BASE_URL || '';
```

Comment:
Since there is no state, I would say a class is unnecessary. Also, I think in React world people tend to favour functions more than classes. 

Mocking and testing is a good point, however there are ways to mock module exports as well. But if we would want to write a component tests I'd pass these functions as properties, rather than wrapping them into a class.

Reply:
Yeah I’m not close to the javascript community enough to know which side wins. I came purely from a hexagonal architecture point of view and felt like encapsulate similar responsibilities inside a single injectable entity that helps blackbox testing. Again, this is just my personal preference and we are not doing any frontend testing so no need to switch back to have a class wrapper. 

Regarding the functions in this file, do you feel strongly about using arrow functions vs using `function` keyword?